tests/stress/scopez_server/scopez_stress_test.py

```diff
@@ -0,0 +1,279 @@
+            a_vector_file, type(l_e), l_e, l_e.__doc__))
+        sys.exit(-1)
+    l_time_ms_last = time.time()*1000
+    i_c = 0
+    while g_run:
+        i_c += 1
+        post_config(l_host, l_template, a_type, i_c)
+        l_time_ms_cur = time.time()*1000
+        if l_time_ms_cur > (l_time_ms_last + 100):
+            l_time_ms_last = time.time()*1000
+            l_time_ms_next = l_time_ms_last + 100
+            print('%6.2f done'%((((float(i_c))) / (a_num_ids)) *100.0))
+        if i_c == a_num_ids:
+            break
+    if not g_run:
+        return
+    print_banner()
+    # ------------------------------------------------------
+    # read vector file
+    # ------------------------------------------------------
+    l_vectors = []
+    try:
+        with open(a_vector_file) as l_f:
+            l_vectors = json.load(l_f)
+    except Exception as l_e:
+        print('error opening vector file: %s.  Reason: %s error: %s, doc: %s' % (
+            a_vector_file, type(l_e), l_e, l_e.__doc__))
+        sys.exit(-1)
+    # print(l_vectors)
+    # ------------------------------------------------------
+    # setup
+    # ------------------------------------------------------
+    l_v_size = len(l_vectors)
+    l_v_idx = 0
+    l_time_ms_last = time.time()*1000
+    l_num_reqs = 0
+    l_num_reqs_total = 0
+    l_num_confs = 0
+    l_num_confs_total = 0
+    l_results = {}
+    # ------------------------------------------------------
+    # run...
+    # ------------------------------------------------------
+    while g_run:
+        l_id = random.randint(1, a_num_ids)
+        get_rqst(l_host, l_id, l_vectors, l_v_idx, l_results)
+        l_v_idx += 1
+        if l_v_idx >= l_v_size:
+            l_v_idx = 0
+        l_num_reqs += 1
+        l_num_reqs_total += 1
+        if l_num_reqs_total % 100 == 0:
+            post_config(l_host, l_template, a_type, int(l_id))
+            l_num_confs += 1
+            l_num_confs_total += 1
+        l_time_ms_cur = time.time()*1000
+        if l_time_ms_cur > (l_time_ms_last + 100):
+            print_stats_line(l_time_ms_cur - l_time_ms_last, l_num_reqs, l_num_confs, l_results)
+            l_time_ms_last = time.time()*1000
+            l_time_ms_next = l_time_ms_last + 100
+            l_num_reqs = 0
+            l_num_confs = 0
+            l_results = {}
+    # ------------------------------------------------------
+    # done...
+    # ------------------------------------------------------
+    print('...shutting down...')
+# ------------------------------------------------------------------------------
+# main
+# ------------------------------------------------------------------------------
+def main(argv):
+    l_arg_parser = argparse.ArgumentParser(
+        description='scopez_server stress tester.',
+        usage='%(prog)s <any one template file(acl, rules, profile, scopes, limit), template type and request vector file',
```

Comment:
I think it would be more clear like this:
`usage='%(prog)s -t <any one template file (acl, rules, profile, scopes, limit)> -a <template type> -x <request vector file>',`